### ISSUE TITLE
After updating to 0.15.7, tag tags can't be added properly.

### ISSUE THREAD

**Issue by jerry-f** [toeverything/AFFiNE#7595]
### What happened?

After updating to version 0.15.7, it is not possible to add tags properly.

The console has an alert error message when the tab is opened:

```
react-dom.development.js:86 Warning: Received `false` for a non-boolean attribute `focused`.

If you want to write it to the DOM, pass a string instead: focused="false" or focused={value.toString()}.

If you used to conditionally omit it with focused={condition && value}, pass focused={condition ? value : undefined} instead.
    at div
    at div
    at div
    at eval (webpack-internal:///../../../node_modules/@radix-ui/react-primitive/dist/index.mjs:35:13)
    at eval (webpack-internal:///../../../node_modules/@radix-ui/react-scroll-area/dist/index.mjs:112:13)
    at _c2 (webpack-internal:///../component/src/ui/scrollbar/scrollable.tsx:36:101)
    at div
  ....

```

### Distribution version

Windows x64

### Are you self-hosting?

- [ ] Yes

### Relevant log output

```shell
Browser name and version: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
```

### Anything else?

Browser name and version

**Comment by jerry-f** [comment]
You guys didn't notice this problem, have you fixed it yet?

**Comment by JimmFly** [comment]
Apologies for the delayed response. We are currently addressing this issue and it will be fixed in an upcoming stable release.

**Comment by forehalo** [comment]
fixed in 0.16.0
